WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -436,7 +435,42 @@ public void onErrorResponse(VolleyError error) {
                 });
     }
 
+    private void fetchJetpackMonitorSettings() {
+        WordPress.getRestClientUtils().getJetpackMonitor(
+                mSite.getSiteId(), new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        mRemoteJpSettings.monitorActive = response.optBoolean("active");
+                        mJpSettings.monitorActive = mRemoteJpSettings.monitorActive;
+                        notifyUpdatedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error getting Jetpack Monitor settings: " + error);
+                    }
+                });
+    }
+
+    private void updateJetpackMonitorSettings() {
+        WordPress.getRestClientUtils().setJetpackMonitor(
+                mSite.getSiteId(), new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        mRemoteJpSettings.monitorActive = response.optBoolean("active");
+                        mJpSettings.monitorActive = mRemoteJpSettings.monitorActive;
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack Monitor settings: " + error);
+                    }
+                }, mJpSettings.monitorActive);
+    }
+
     private void saveJetpackSettings() {
+        updateJetpackMonitorSettings();
```

Comment:
If we are calling the `updateJetpackMonitorSettings` from `saveJetpackSettings`, should we also call `fetchJetpackMonitorSettings` from `fetchJetpackSettings` instead of from the callback?